firestore/query.go

```diff
@@ -392,8 +392,8 @@ const (
 	DistanceMeasureDotProduct DistanceMeasure = DistanceMeasure(pb.StructuredQuery_FindNearest_DOT_PRODUCT)
 )
 
-// FindNearestOpts is options to use while building FindNearest vector query
-type FindNearestOpts struct {
+// FindNearestOptions is options to use while building FindNearest vector query
```

Comment:
"are options for a FindNearest vector query"

Reply:
Updated